### DISCUSSION TITLE
How do I make a link so that I come to the login page?

### DISCUSSION THREAD

**Discussion by BogusCurry** [datenstrom/community#755] [Ask a question] [answered]
Hello all ;D
This is my first website with Yellow, well which I plan in the long term, Yellow is already a very nice CMS. But have a problem, namely the login, can I put a link on the website, how can I put a link on the startsite, so that come to the login page ? 

Greetings

Bogus

**Comment by IngoBelka** [comment, marked as answer]
Look and analyze this demo page
[Yellow demo try](https://datenstrom.se/yellow/help/troubleshooting)
There you find a link to a login/edit page.

**Reply by annaesvensson** [reply]
Here's an example from the Datenstrom Yellow demo:

```
---
Title: Try the demo
TitleNavigation: Demo
TitleSlug: demo
Layout: demo
---
[image photo.jpg Example rounded]

[edit - You can edit this page] or use a text editor.
```

Here's another example with a link to the login page:

```
---
Title: Example page
---
Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut 
labore et dolore magna pizza. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris 
nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit 
esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt 
in culpa qui officia deserunt mollit anim id est laborum.

[edit - Go to the login page].
```

To show an edit link on your website use an `[edit]` shortcut. Easy peasy lemon squeezy. 😄

**Comment by BogusCurry** [comment]
Ok, the answer is very short, put [edit] on the website ;D

**Reply by ghost** [reply]
@BogusCurry  this means I can create any page I want (like login.php', one that only I know, and use that as a login page by inserting that '[edit]' ? Right now, when I upload yellow onto a server, it seems anyone can set up an account if they like. I cannot create just one user and be the sole maintainer. Instead, I get a little party going. Any way to have just one maintainer? For instance, can I update locally and then upload the files? That way I need not bother with logging in? Just modify and upload as needed?
